contracts/Interfaces.sol

```diff
@@ -152,6 +152,8 @@ contract Interfaces is Admin, Initializable {
     _chainIdMap[ChainIdType.EVM][250][ChainIdType.HOLOGRAPH] = 5;
     // rinkeby
     _chainIdMap[ChainIdType.EVM][4][ChainIdType.HOLOGRAPH] = 4000000001;
+    // goerli
+    _chainIdMap[ChainIdType.EVM][5][ChainIdType.HOLOGRAPH] = 4000000011;
```

Comment:
Are these Holograph IDs arbitrary?

Reply:
There is some logic to the numbers, 4294967295 is max value for 2^32 which is uint32 (our highest chain id value).
We use 4294967295 and 4294967294 for localhost network ids.
We reserve all numbers 4000000001 and up for testnets.
Mainnets start from 1 up to 4000000000.
As we identify/use new testnets or mainnets, we assign them the next available Holograph ChainID. Goerli, being a testnet, was assigned 4000000011, since that was the next available number when we were adding it to the list.

Reply:
aha okay cool. thanks for explaining 👏